src/Turn.js

```diff
@@ -1,19 +1,15 @@
-const Card = require('../src/Card');
-
 class Turn {
-  constructor() {
+  constructor(guess, card) {
+    this.guess = guess;
+    this.card = card;
```

Comment:
Instead of passing arguments in to each method from the round class, this is now instantiated will the relevant information.